### PR TITLE
Loc counter

### PR DESCRIPTION
**Pull request by giulianobelinassi**
Added Lines of Code metric.

### CODE REVIEW

File: spec/parser/xml_builder_engine_spec.rb [hunk 1]
```diff
@@ -248,6 +248,20 @@
...
+        linesOfCode counter: "42"
+      end
+
+      output = [@basicHeader,
+                '<kuniri>',
+                '  <linesOfCode counter="42"/>',
+                '</kuniri>'].join("\n") + "\n"
+      expect(@builder.to_xml).to eq(output)
+    end
+  end
```
Comment (rodrigosiqueira) [inline spec/parser/xml_builder_engine_spec.rb]: There is an extra whitespace in the end of 'end' in line 264

**Comment by rodrigosiqueira** [comment]
Almost done, just miss two things:
1. The extra space at the end of the spec file (refer to the review comment)
2. When we execute kuniri with O1 flag it optimizes the final output to use a reduced version of the elements name. lineOfCode could be reduced to loc and count to c when -O1 is passed to the CLI.
Thanks for your contribution :)

**Comment by rodrigosiqueira** [comment]
I just miss some tests similar the ones find in spec/language/ruby/global_test/. Add rspec tests with multiple variation of this parameter and other similar with tests in spec/language/ruby/global_test/

**Comment by ghost** [comment]
We've implemented the tests and commited the changes to the same branch.

**Pull request merged**